Sources/TuistGenerator/Templates/AssetsTemplate.swift

```diff
@@ -163,10 +163,20 @@ extension SynthesizedResourceInterfaceTemplates {
       }()
 
       #if canImport(SwiftUI)
+      private var _swiftUIColor: Any? = nil
       @available(iOS 13.0, tvOS 13.0, watchOS 6.0, macOS 10.15, *)
-      {{accessModifier}} private(set) lazy var swiftUIColor: SwiftUI.Color = {
-        SwiftUI.Color(asset: self)
-      }()
+      {{accessModifier}} private(set) var swiftUIColor: SwiftUI.Color {
+        get {
+          if self._swiftUIColor == nil {
+            self._swiftUIColor = SwiftUI.Color(asset: self)
+          }
+
+          return self._swiftUIColor as! SwiftUI.Color
+        }
+        set {
+          self._swiftUIColor = newValue
+        }
+      }
       #endif
```

Comment:
@fortmarek @kwridan do you think there is a better way to do this? :D

Reply:
Oof, not the prettiest but I guess the only good alternative would be to make `swiftUIColor` a method which would be breaking.